src/Tickets/Emails/Legacy.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace TEC\Tickets\Emails;
+
+use Tribe__Tickets__Tickets as Tickets_Module;
+
+/**
+ * Class Legacy
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Legacy {
```

Comment:
Calling this `Legacy` does feel weird. Perhaps change it to what it is actually doing: `Legacy_Hijack` or `Legacy_Hijacker`. That'd make it clear that this code is meant to override legacy logic and isn't, itself, legacy logic.